stable/redis-ha/templates/NOTES.txt

```diff
@@ -0,0 +1,12 @@
+Redis cluster can be accessed via port 6379 on the following DNS name from within your cluster:
+redis-svc.{{ .Release.Namespace }}.svc.cluster.local
+
+To connect to your Redis server:
+
+1. Run a Redis pod that you can use as a client:
+
+   kubectl exec -it redis-master bash
```

Comment:
This needs to be templatized so that it matches what is actually deployed in the cluster. Example here:
https://github.com/kubernetes/charts/blob/master/stable/prometheus/templates/NOTES.txt#L22